README.md

```diff
@@ -43,7 +43,7 @@ The command above will display a list of available options. Make sure you read [
 
 ### Create a migration file with tables from your database
 
-To create your first migration file, execute:
+If you do not want to write your first migration file by hand, you can generate one using the [cakephp/bake](https://github.com/cakephp/bake) plugin:
```

Comment:
Not fond of this - who _does_ want to write their first migration file by hand? You'd never do that, and written this way suggests it's normal and possibly even a good idea.